packages/flutter_tools/lib/src/base/fingerprint.dart

```diff
@@ -62,7 +62,9 @@ class Fingerprinter {
   void writeFingerprint() {
     try {
       final Fingerprint fingerprint = buildFingerprint();
-      _fileSystem.file(fingerprintPath).writeAsStringSync(fingerprint.toJson());
+      final File fingerprintFile = _fileSystem.file(fingerprintPath);
+      fingerprintFile.createSync(recursive: true);
+      fingerprintFile.writeAsStringSync(fingerprint.toJson());
```

Comment:
```
Fingerprint write error: FileSystemException: Cannot open file, path = 'build/ios/pod_inputs.fingerprint' (OS Error: No such file or directory, errno = 2)
```
`writeAsStringSync` isn't recursively creating the directory structure.
https://github.com/flutter/flutter/blob/ca6cecf034d9c18fcb6ea70309cc855e353aa0f8/packages/flutter_tools/lib/src/base/fingerprint.dart#L64-L65

Reply:
d'oh